src/mon/OSDMonitor.cc

```diff
@@ -11262,6 +11262,9 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     bool force = false;
     cmd_getval(cmdmap, "force", force);
 
+    bool sure = false;
+    cmd_getval(cmdmap, "yes_i_really_mean_it", sure);
```

Comment:
How about --allow-experimental-ec-profile

Reply:
I'll look into implementing this, although I think `--yes-i-really-mean-it` communicates the same thing, and is a flag already known to users. Any special reason to create a new flag?

Reply:
Sam explained, we do want an extra flag to make users basically type out what they're doing to ensure understanding there. Will work on this.

Reply:
Added.